### ISSUE TITLE
Ticking Entity Server Crash

### ISSUE THREAD

**Issue by Jollyolsport** [seymourimadeit/guardvillagers#27]
Latest log; they seem to have trouble targeting mobs outside of vanilla and/or zombies? 
[latest.log](https://github.com/seymourimadeit/guardvillagers/files/5230939/latest.log)

**Comment by ColdFang** [comment]
I am getting similar crashes with 1.16.1 version. I did a little bit of testing; with just guard villagers installed i cant reproduce the crash - 20 guards vs 20 zombies fighting just fine. 
I installed AI Improvements because it was my first suspect and then the crashes happen when I spawn like 5 zombies in a village. 

[latest.log](https://github.com/seymourimadeit/guardvillagers/files/5239104/latest.log)
